prebuild/resources/addons.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
why do we need to include any manifest in terraform?

Reply:
yes, that's the original idea. But I can remove the manifest templates from this PR so we can rewrite them using tags.

Reply:
this should be just a minimum infra that allows us to test how to create a simple machineset using tags.
I might be mistaken but I don't see the need to generate manifests dynamically
- Manifests for creating the cluster api stack are known ahead of time
- Manifests for machineSet use cases are known ahead of time
If we need to generate something dynamically in the future we can always add it. Let's get the simple case working first